src/Features/Core/Portable/GenerateConstructorFromMembers/GenerateConstructorFromMembersCodeRefactoringProvider.cs

```diff
@@ -46,7 +48,8 @@ internal GenerateConstructorFromMembersCodeRefactoringProvider(IPickMembersServi
                 return;
             }
 
-            var actions = await this.GenerateConstructorFromMembersAsync(document, textSpan, cancellationToken).ConfigureAwait(false);
+            var actions = await this.GenerateConstructorFromMembersAsync(
+                document, textSpan, addNullChecks: false, cancellationToken: cancellationToken).ConfigureAwait(false);
             context.RegisterRefactorings(actions);
```

Comment:
This line makes it look like we've unconditionally registered a refactoring. Can you check IsDefaultOrEmpty before registering the refactoring instead of using this pattern?